vpr/src/route/rr_graph_reader.cpp

```diff
@@ -69,7 +69,7 @@ void load_rr_file(const t_graph_type graph_type,
         &device_ctx.rr_indexed_data,
         device_ctx.num_arch_switches,
         device_ctx.arch_switch_inf,
-        device_ctx.rr_segments,
+        device_ctx.rr_graph_builder.rr_segments(),
```

Comment:
@tangxifan 
I have used rr_grapg_builder here than its passing unit and strong tests. Yesterday you said to use rr_graph_view here than it was failing.Can you Please have a look?